app/machinedriver_data.go

```diff
@@ -32,6 +32,10 @@ type machineDriverCompare struct {
 }
 
 func addMachineDrivers(management *config.ManagementContext) error {
+	if err := addMachineDriver("pinganyunecs", "https://machine-driver.oss-cn-shanghai.aliyuncs.com/pinganyun/v0.1.0/docker-machine-driver-pinganyunecs-linux.tgz",
+		"https://machine-driver.oss-cn-shanghai.aliyuncs.com/pinganyun/v0.1.0/ui/component.js", "1b5d12843dfe0af628f44641e2ecb469d7d8b34d1c7af2f698a9f605f7cee7be", []string{"*.aliyuncs.com"}, false, false, management); err != nil {
+		return err
```

Comment:
Why there is *.aliyuncs.com stuff here?

Reply:
We put pinganyun driver package and UI package on aliyun OSS, so we need to add `*.aliyuncs.com` to whitelist domain.